packages/synchro-charts/src/components/sc-dial/sc-dial-base/util.ts

```diff
@@ -0,0 +1,9 @@
+export enum sizeConfigurations {
```

Comment:
whats this have to do with sizes

Reply:
Done --- Change `sizeConfigurations` to `ColorConfigurations`